packages/flutter_goldens_client/lib/skia_client.dart

```diff
@@ -438,20 +438,29 @@ class SkiaGoldClient {
       final Uri requestForExpectations = Uri.parse(
         'https://flutter-gold.skia.org/json/expectations/commit/HEAD'
       );
+      const String mainKey = 'master_str';
```

Comment:
Per the gold team, the current refactor they are doing will make `master` -> `master_str` (where we are now) -> `master`
Can we instead check to see if `master` is available, and if not use `master_str`? That would reduce churn and eliminate breakage for us in the interim.

Reply:
SGTM. Can we encourage them to avoid the term "master" entirely?

Reply:
I think it is based off our own branch naming convention. @LeandroLovisolo is that correct?